pkg/operator/k8sutil/configmap.go

```diff
@@ -43,17 +46,16 @@ func DeleteConfigMap(clientset kubernetes.Interface, cmName, namespace string, o
 // returns defaultValue if setting is not found
 func GetOperatorSetting(clientset kubernetes.Interface, settingName, defaultValue string) (string, error) {
```

Comment:
The methods in the k8sutil package are intended to be general for any storage provider. Here are some suggestions to generalize this method:
- The configmap name should be an argument to the method. Then different storage providers could pass their own configmap name (instead of `rook-ceph-operator-config`).
- Define the configmap name in `pkg/operator/ceph/cluster/controller.go`. Instead of `RookCephOperatorSetting` I'd suggest `OperatorSettingConfigMapName`.

Reply:
> The methods in the k8sutil package are intended to be general for any storage provider. Here are some suggestions to generalize this method:
> 
>     * The configmap name should be an argument to the method. Then different storage providers could pass their own configmap name (instead of `rook-ceph-operator-config`).
> 
Done.

>     * Define the configmap name in `pkg/operator/ceph/cluster/controller.go`. Instead of `RookCephOperatorSetting` I'd suggest `OperatorSettingConfigMapName`.
>
This will bring cyclic import. `Cluster` imports `csi` and `csi` imports `cluster` (to request for config).
Is there any other place where we can keep this variable?

Reply:
> This will bring cyclic import. Cluster imports csi and csi imports cluster (to request for config).
Is there any other place where we can keep this variable?

How about in the `pkg/operator/ceph/controller` package? It is intended to provide helpers for all the ceph operator.

Reply:
This works.